app/submitter/submitter.py

```diff
@@ -86,7 +86,8 @@ def send_message(self, message, queue):
         :return: a boolean value indicating if it was successful
         """
         message_as_string = str(message)
-        logger.info("sending message to rabbit mq", message=message_as_string)
+        logger.info("sending message to rabbit mq")
+        logger.info("message payload", message=message_as_string)
```

Comment:
Discussing with @eldeal it would be good to have a 'category='RabbitMQ'` or similar